.travis.yml

```diff
@@ -59,7 +59,7 @@ install:
 
 # now run the tests!
 script:
-  - grunt nsp --force # check for vulnerable modules via nodesecurity.io
+  - grunt nsp # check for vulnerable modules via nodesecurity.io
```

Comment:
That's pretty much the default behavior for Grunt tasks (ie: all the linting tasks pretty much fail on errors).